src/main/java/greencity/entities/Location.java

```diff
@@ -0,0 +1,28 @@
+package greencity.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Location {
```

Comment:
I think it is better to not have Location a separate table in the database for location but either put latitude and longitude fields to the Place entity (Useless OneToOne relation). These fields will be used almost always so maybe better remove this relation?